pkg/codegen/internal/test/testdata/simple-resource-schema/otherResource.go

```diff
@@ -0,0 +1,68 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+package example
+
+import (
+	"reflect"
+
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+type OtherResource struct {
+	pulumi.CustomResourceState
```

Comment:
If it's a component, I believe this should be `pulumi.ResourceState`.

```suggestion
	pulumi.ResourceState
```